sentry_sdk/integrations/sanic.py

```diff
@@ -150,6 +150,9 @@ def _capture_exception(exception):
     # If an integration is there, a client has to be there.
     client = hub.client  # type: Any
 
+    if client and exception in client.ignore_errors:
```

Comment:
Not entirely sure how this works, do you have exception instances in `ignore_errors`?